addons/website/static/src/snippets/s_table_of_content/options.js

```diff
@@ -175,6 +175,8 @@ options.registry.TableOfContentMainColumns = options.Class.extend({
      * @override
      */
     start: function () {
+        // This cannot be replaced by using oe_unremovable, because it would
+        // make the editor forbid moving the navbar using the navbarPosition.
```

Comment:
I am not sure this comment is relevant with what this commit does :thinking:  I think it should rather be an answer to BVR's comment or a note in a task dedicated to improve this part of the code :wink: